src/Ookii.Dialogs.WinForms/InputDialog.cs

```diff
@@ -39,6 +39,7 @@ public partial class InputDialog : Component, IBindableComponent
         private string _input;
         private int _maxLength = Int16.MaxValue;
         private bool _usePasswordMasking;
+        private bool _multiLine = false;
```

Comment:
Initializing a `bool` with `false` is not needed (because it's the default), so you can remove it, so it should be:

```csharp
private bool _multiLine;
```